1.35.0 array blog post (#3344)
PostHog/posthog.com: contents/blog/the-posthog-array-1-35-0.md

---
date: 2022-04-25

> title: Array 1.35.0

Reviewer: Could we starting giving these more of a headline? Something like 'Array 1.35.0: Introducting xyz'

Reviewer 2: Aye, this is something we've done for the last few emails and would be a good add here. Will put commit something in, but happy to tweak.

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["Release notes", "Product updates"]
featuredImage: ../images/blog/posthog-array-blog.png
featuredImageType: full
excerpt: PostHog 1.35.0 introduces activity logs and a brand new way of visualizing where your users are coming from with the World Map. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus check out your Project Homepage for a few additional goodies.
---

> PostHog 1.35.0 introduces audit logs and a brand new world map view to help you visualize where your users are coming from. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus, check out your project homepage for a few additional goodies!

Reviewer: ```suggestion
PostHog 1.35.0 introduces audit logs and a brand new world map view to help you visualize where your users are coming from. Additionally, we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus, check out your project homepage for a few additional goodies!
```

We've also recently launched a new newsletter, which we send once every two weeks! [Subscribe today](/newsletter) to hear about our latest news, what we're reading and how to get the most out of PostHog. Each issue also includes a bonus hedgehog cartoon!

<blockquote class='warning-note'>
<b>IMPORTANT!</b> If you use SAML, please read our <a href="deprecation-and-removal-notices">deprecation notices below</a>.
</blockquote>

## PostHog 1.35.0 release notes

> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).

**Release highlights:**
- [Activity log](#new-activity-log)
- [World map](#new-world-map)
- [Multi-tenant SAML](#new-multitenant-saml)
- [Project homepage updates](#improvement-project-homepage-updates)
- [New Avo plugin](#new-avo-plugin)

### New: Activity log
<img width="827" alt="A screenshot of the app showing the new 'history' tab on the feature flags page. In the screenshot you can see three recent changes by PostHog staff to our feature flags" src="https://user-images.githubusercontent.com/984817/164484091-f5dfce0b-c400-4699-ac55-642ccc2bc55b.png" />

> Ever wondered who deleted that feature flag, or merged those two people? Wonder no more with the new Activity Log™️!

Reviewer 2: ```suggestion
Ever wondered who deleted that feature flag, or merged those two people? Wonder no more with the new activity log!
```

Charles has asked us to remove the TM.

Reviewer 3: @charlescook-ph can we say "patent pending" :)

Reviewer 4: haha, oh Paul you joker

but no, we can't 🧑‍⚖️ 🚓

Reviewer 3: 😢

---
date: 2022-04-25
title: Array 1.35.0
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["Release notes", "Product updates"]
featuredImage: ../images/blog/posthog-array-blog.png
featuredImageType: full

> excerpt: PostHog 1.35.0 introduces audit logs and a brand new way of visualizing where your users are coming from with the World Map. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus check out your Project Homepage for a few additional goodies.

Reviewer 2: ```suggestion
excerpt: PostHog 1.35.0 introduces activity logs and a brand new way of visualizing where your users are coming from with the World Map. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus check out your Project Homepage for a few additional goodies.
```

---

PostHog 1.35.0 introduces audit logs and a brand new World Map view to help you visualize where your users are coming from. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus, check out your project homepage for a few additional goodies!

We've also recently launched a new newsletter, which we send once every two weeks! Subscribe today to hear about our latest news, what we're reading and how to get the most out of PostHog. Each issue also includes a bonus hedgehog cartoon!

<blockquote class='warning-note'>
<b>IMPORTANT!</b> If you use SAML, please read our <a href="deprecation-and-removal-notices">deprecation notices below</a>.
</blockquote>

## PostHog 1.35.0 release notes

> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).

**Release highlights:**

> - [Activity Log](#new-activity-log)

Reviewer 2: ```suggestion
- [Activity log](#new-activity-log)
```

- [World Map](#new-world-map)
- [Multitenant SAML](#new-multitenant-saml)
- [Project Homepage Updates](#improvement-project-homepage-updates)
- [New Avo plugin](#new-avo-plugin)

> ### New: Activity Log

Reviewer 2: ```suggestion
### New: Activity log
```

<img width="827" alt="Screenshot 2022-04-21 at 15 45 07" src="https://user-images.githubusercontent.com/984817/164484091-f5dfce0b-c400-4699-ac55-642ccc2bc55b.png">

Ever wondered who deleted that feature flag, or merged those two people? Wonder no more with the new Activity Log™️!

> You can now view changes to feature flags and persons made in the UI and see who made them, when. Watch out for the Activity Log being added to more pages in future releases. Or join [our community Slack](https://posthog.com/community) and tell us where we should add it next.

Reviewer 2: ```suggestion
You can now view changes to feature flags and persons made in the UI and see who made them, when. Watch out for the activity log being added to more pages in future releases. Or join [our community Slack](https://posthog.com/community) and tell us where we should add it next.
```

> You can now view changes to feature flags and persons made in the UI and see who made them, when. Watch out for the activity log being added to more pages in future releases. Or join [our community Slack](https://posthog.com/community) and tell us where we should add it next.

Reviewer 2: ```suggestion
You can now view changes to feature flags and persons made in the UI and see who made them, when. Watch out for the activity log being added to more pages in future releases. Or join [our community Slack](https://posthog.com/slack) and tell us where we should add it next.
```

### New: World Map
![PostHog - The World Map](../images/blog/array/1_35_0-map.png)

A staple request of every PostHog hackathon has finally been realized, with the introduction of a new World Map chart view, which enables you to visualize locational data easily. No more having to work out popular territories via a table view!

The World Map is obviously especially useful for finding out where your users physically are, so you can do things like comparing the difference between your traffic and sign-ups. 

> ### New: Multitenant SAML

Reviewer 2: ```suggestion
### New: Multi-tenant SAML
```

[[TODO: @camerondeleone do you have a screenshot that could go here about this feature? https://github.com/PostHog/posthog/pull/9225]]

SAML has been a big focus point in this update (check the deprecation notices below for more info) and the upshot is that you can now configure SAML login on both self-hosted and PostHog Cloud instances! 

You can configure up to one SAML provider per organization, but check out our [SSO docs](/sso) for more information on how to get started with SAML on PostHog. 

<blockquote class='warning-note'>
<b>IMPORTANT!</b> If you use SAML please read our <a href="deprecation-and-removal-notices">deprecation notices below</a>. SAML support is only available on Enterprise plans.
</blockquote>

### Improvement: Project homepage updates
![PostHog - Project Homepages](../images/blog/array/1_34_0-homepage.png)

> In the last update, we introduced [project homepages](https://posthog.com/blog/the-posthog-array-1-34-0#new-homepage) to help you find information quickly. Now, we've made them even better!

Reviewer 2: ```suggestion
In the last update, we introduced [project homepages](blog/the-posthog-array-1-34-0#new-homepage) to help you find information quickly. Now, we've made them even better!
```

We've added a new homepage card which lists your recently viewed insights to the project homepage, as well as lists of recent recordings and newly identified persons. All of this is to help give you quick access to commonly-visited pages and recent insights.

### New: Avo plugin
![PostHog - Avo plugin](../images/blog/array/1_35_0_avo.png)

We're big fans of [the Avo data governance platform](https://www.avo.app/) and were lucky enough to meet the team in person at our recent Iceland offsite. Now, we've launched a new plugin too!

> The Avo plugin sends events to the Avo Inspector as they are ingested by PostHog, enabling you to detect inconsistencies in your product data and help you understand how your tracking is performing. Install it from the plugin tab to get started, or find out more in [Avo's documentation](https://www.avo.app/docs/workspace/connect-inspector-to-posthog#step-2-enable-avo-inspector-plugin-in-posthog). 

Reviewer 2: ```suggestion
The plugin sends event schemas to the Avo Inspector as they are ingested by PostHog, enabling you to detect inconsistencies in your schema without sharing any user data. Install it from the plugin tab to get started, or find out more in [Avo's documentation](https://www.avo.app/docs/workspace/connect-inspector-to-posthog#step-2-enable-avo-inspector-plugin-in-posthog). 
```

featuredImageType: full
excerpt: PostHog 1.35.0 introduces audit logs and a brand new way of visualizing where your users are coming from with the World Map. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus check out your Project Homepage for a few additional goodies.
---

PostHog 1.35.0 introduces audit logs and a brand new way of visualizing where your users are coming from with the World Map. Additionally we now support organization-level SAML login on both Cloud and Self-Hosted instances. Plus check out your Project Homepage for a few additional goodies.

<blockquote class='warning-note'>
<b>IMPORTANT!</b> If you use SAML please read our <a href="deprecation-and-removal-notices">deprecation notices below</a>.
</blockquote>

## PostHog 1.35.0 release notes

> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).

**Release highlights:**
- [Activity Log](#new-activity-log)
- [World Map](#new-world-map)
- [Multitenant SAML](#new-multitenant-saml)
- [Project Homepage Updates](#improvement-project-homepage-updates)

### New: Activity Log

> [[TODO: screenshot goes here]]

Reviewer 3: ```suggestion
<img width="827" alt="Screenshot 2022-04-21 at 15 45 07" src="https://user-images.githubusercontent.com/984817/164484091-f5dfce0b-c400-4699-ac55-642ccc2bc55b.png">
```

> [[TODO: @pauldambra Could you give a brief spiel about activity log here?]]

Reviewer 3: ```suggestion
Ever wondered who deleted that feature flag, or merged those two people? Wonder no more with the new Activity Log™️!

You can now view changes to feature flags and persons made in the UI and see who made them. Watch out for the activity log being added to more pages in future releases. Or join [our community Slack](https://posthog.com/community) and tell us where we should add it next.
```
